tests/examples/awful/placement/align.lua

```diff
@@ -0,0 +1,25 @@
+--DOC_HIDE_ALL
+local placement = require("awful.placement")
+screen[1]._resize {x= 50}
```

Comment:
The shims added by the other PR have some `_something` methods like those mostly for the screen stuff.